includes/blocks/class-sensei-global-blocks.php

```diff
@@ -32,7 +32,6 @@ public function initialize_blocks() {
 		new Sensei_Course_Progress_Block();
 		new Sensei_Course_Overview_Block();
 		new Sensei_Course_List_Block();
-		new Sensei_Course_Actions_Block();
```

Comment:
Shouldn't we also keep this one?

Reply:
I don't think so as per my comment in the description:

> Although I haven't confirmed, I'm suspicious that this change may have also introduced an issue where the View Certificate button is no longer displayed as per 8590896-zd-a8c. So I'm reverting that change as well.

Reply:
Now that the translations / question description issue is fixed, I'll confirm this today.

Reply:
After reviewing the latest details in the ticket, it appears the issue they are having is site-specific and not a regression, so the fact that they reported it the day after we did the release is just a coincidence. 😄  Given this, I've restored the changes related to the Course Actions block. So then, all this PR does is fix the question description not being displayed. I've updated the title to reflect that. 🙂 Thanks!